qa/L0_metrics/test.sh

```diff
@@ -111,35 +111,60 @@ mkdir -p $MODELDIR/${model}/1 && \
   sed -i "s/label_filename:.*//" config.pbtxt && \
   echo "instance_group [{ kind: KIND_GPU }]" >> config.pbtxt)
 
+### CPU / RAM metrics tests
+set +e
+SERVER_LOG="cpu_metrics_test_server.log"
+# NOTE: CPU utilization is computed based on the metrics interval, so having
+# too small of an interval can skew the results.
+SERVER_ARGS="$BASE_SERVER_ARGS --metrics-interval-ms=1000 --log-verbose=1"
+run_and_check_server
+
+CLIENT_PY="./cpu_metrics_test.py"
+CLIENT_LOG="cpu_metrics_test_client.log"
+python3 -m pytest --junitxml="cpu_metrics.report.xml" ${CLIENT_PY} >> ${CLIENT_LOG} 2>&1
+if [ $? -ne 0 ]; then
+    echo -e "\n***\n*** ${CLIENT_PY} FAILED. \n***"
```

Comment:
What's our rule regarding whether we `cat` the client/server logs on failure? I see mixed usage across our tests, but think having failure logs built-in to the pipeline regardless of artifact expiration is valuable.

Reply:
Nice shout - I'm pro dumping logs to console in case artifacts expire.